Project/webpack.config.js

```diff
@@ -86,6 +87,7 @@ module.exports = {
         allowedHosts:[
             '.azurewebsites.net'
         ],
+        https: true,
```

Comment:
is this needed?

Reply:
Removed. Not needed anymore. Was a remnant of my proxy server experiments :D